app/controllers/companies_controller.rb

```diff
@@ -96,6 +99,7 @@ def company_params
     params.require(:company).permit(:name, :company_number, :phone_number,
                                     :email,
                                     :website,
+                                    :info,
```

Comment:
I know this will be a bear to change, but `info` is way too generic.  Why not something like "customized_description"  (or "customized_desc" if you want a shorter version).